wordpress-mobile/WordPress-iOS
# Issue/8536 track media error messages in db

Fixes #8536 

To test:
 - Start a new post in aztec
 - Try to add/upload a media object
 - Make it fail on the middle of the process ( turn of connectivity for example)
 - See if attachment switches to the correct state
 - Check if actions on media are correct by tapping on it
 - Enable connectivity again
 - Retry the upload of media
 - Check if uploads correctly and status is updated correctly.
 - On the media library, make an upload fail also
 - Tap on media that failed, check if alert show message with the details of why the upload failed.
